### ISSUE TITLE
Publication Listings & Preprint Links

### ISSUE THREAD

**Issue by nelsonni** [EPICLab/EPICLab.github.io#28]
The [Publications](https://epiclab.github.io/publications.html) page is expected to host links to the official publication, but non-institution affiliated visitors are often barred from accessing these publications due to publishers' paywall-restrictions. However, preprint versions of the publications can be hosted under most academic publishers' [preprint policies](https://en.wikipedia.org/wiki/List_of_academic_journals_by_preprint_policy).

Our current publication listings appear in the following format:
<img src="https://user-images.githubusercontent.com/6432572/91202043-1a6cdd80-e6b6-11ea-87b4-5f857f5163f7.png" width="700">

In order to accommodate preprint links (as well as providing additional reference information and awards indicators) we will switch to the following format for all publications (therefore, this format supersedes the format described in #16):
```yaml
{{Venue Acronym}}  {{Publication Title}}
                   {{Author(s)}}
                   {{Venue Name}} ({{Venue Acronym}}), {{Publisher}}, {{Year}}.
                   {{Awards}}
                   {{DOI}} · {{PDF}} · {{Slides}} · {{Tool}} · {{Data}}
```
As an example, the _"A Tale from the Trenches: Cognitive Biases and Software Development"_ publication would appear as:

> [ICSE 2020](https://conf.researchr.org/home/icse-2020)  **A Tale from the Trenches: Cognitive Biases and Software Development**
       [Souti Chattopadhyay](https://epiclab.github.io/people.html), [Nicholas Nelson](https://epiclab.github.io/people.html), Audrey Au, Natalia Morales, [Christopher Sanchez](http://people.oregonstate.edu/~sancchri/), 
       [Rahul Pandita](http://rahulpandita.me/), and [Anita Sarma](https://epiclab.github.io/people.html)
       _International Conference on Software Engineering (ICSE)_, ACM, 2020.
       🏆 ACM SIGSOFT Distinguished Paper Award
       DOI · [PDF](https://nomatic.dev/docs/icse20-chattopadhyay.pdf) · [Data](https://epiclab.github.io/ICSE20-CogBias/)

Any field not available for a given publication should not be displayed (except for **DOI**, which should always be included and converted to a link when the DOI becomes available for a publication). Remember that authors names should conform to [Chicago Citation Style](https://www.scribbr.com/category/chicago-style/), in the format of `{{First Name}} {{Last Name}}`, and that author links should be to the [People](https://epiclab.github.io/people.html) page for current EPICLab members.

**Comment by nelsonni** [comment]
This update is a change from the previous standard of creating publication-specific sites (e.g. https://epiclab.github.io/ICSE20-CogBias/) for individual publications to gather datasets and related links. Going forward, we will include these types of links and files directly on the [Publications](https://epiclab.github.io/publications.html) page using the bottom-row links described above.

However, in the case of the _"A Tale from the Trenches: Cognitive Biases and Software Development"_ publication there is a URL reference to https://epiclab.github.io/ICSE20-CogBias/ within the publication itself:
<img src="https://user-images.githubusercontent.com/6432572/91207878-4f7d2e00-e6be-11ea-8c82-a8ff4f2a65b4.png" width="650">

Therefore, we will preserve this URL and make it available through the **Data** link on the publication listing.

**Comment by nelsonni** [comment]
All preprints should be stored as PDF files in the [`publications/`](https://github.com/EPICLab/EPICLab.github.io/tree/master/publications) directory and named according to the following format:
```yaml
{{Venue Acronym}}{{Publication Year (2-digit format)}}-{{Last Name of First Author}}.pdf
```
As an example, the _"A Tale from the Trenches: Cognitive Biases and Software Development"_ publication is stored as:
> icse20-chattopadhyay.pdf

**Comment by audreyau** [comment]
For journals (e.g. IEEE Transactions on Software Engineering), what should be put under `{{Venue Acronym}}`? The journal names are too long to fit in that space.

![Screen Shot 2020-08-25 at 11 40 23 AM](https://user-images.githubusercontent.com/52758047/91214329-c4a13100-e6c7-11ea-9993-4f4e13075c46.png)

**Comment by nelsonni** [comment]
There are acronyms for the journals as well. Here is a short primer:
| Journal       | Acronym     |
| :------------- | -----------: |
| Empirical Software Engineering | EMSE | 
| IEEE Transactions on Software Engineering | TSE |
| ACM Transactions on Software Engineering and Methodology | TOSEM |
| Information and Software Technology | IST |
| Journal of Systems and Software | JSS |
| International Journal of Software Engineering and Knowledge Engineering | IJSEKE |
| Journal of Computer Supported Cooperative Work | CSCW |
| Journal of Internet Services and Applications | JISA |
| ACM Transactions on Computer-Human Interactions | TOCHI |
| Journal on Information and Software Technology | JIST |
| IEEE Computer _(no acronym)_ | IEEE Computer |

Journals typically publish on a semi-annual (or quarterly) basis, so listing `Empirical Software Engineering 2020` or `EMSE 2020` would not be appropriate in this case. Instead, journal publications should be listed by the journal acronym (i.e. `EMSE`).

**Comment by audreyau** [comment]
When uploading the PDFs for the publications, is there a certain place I should look to find the specific pre-prints or can I just use the PDFs on Anita's website? Also, where do I locate the slides, tools, and data for each paper?

**Comment by nelsonni** [comment]
Yes, any preprints available on Anita's website can be uploaded and linked from the [`publications/`](https://github.com/EPICLab/EPICLab.github.io/tree/master/publications) directory. Also, all of the publications that I've authored in some capacity are available as preprints on my website: https://nomatic.dev/. There are also `slides`, `tools`, and `data` links on my website that can be directly translated to the EPICLab.github.io site.

Typically, the first authors on papers will likely have most of these artifacts (or at least know where they're currently located). This means that papers involving past EPICLab members will definitely be more difficult to track down, and I would suggest asking Anita if she has any of those artifacts. Otherwise, those publications that cannot be located should just be listed without the links.

**Comment by audreyau** [comment]
For the papers that had presentations (e.g.[ICSE 2020: A Tale From The Trenches](https://youtu.be/KgYF-r_c18g)), should we link videos too?

**Comment by nelsonni** [comment]
Good idea! Let's put those in as a link called `Video` (preferably next to the `Slides` link). I'm updating the issue description at the top to include this new link in the format as well.

**Comment by audreyau** [comment]
The publication _Recommending Tasks to Newcomers in OSS Projects: How Do Mentors Handle It?_ has two data sites. One is from [figshare](https://figshare.com/s/e4d1a8a13a9f5db9202d) (this has the interview script) and the other is a [zenodo record](https://zenodo.org/record/3970997#.X0bBSBNKjUY) (this has the codebook). Should I include both links and have two `data` links?

**Comment by nelsonni** [comment]
The preferred response would be that all data for a publication should be contained in [Zenodo](https://zenodo.org/) or [figshare](https://figshare.com/), but not both.

However, since both sites are providing permanent DOI links for inclusion within the publication themselves, let's create a sub-page that contains both links (and a brief explanation for what each link provides). This can be similar to the https://epiclab.github.io/ICSE20-CogBias/ page created for the _"A Tale from the Trenches: Cognitive Biases and Software Development"_ publication, but contained within the https://github.com/EPICLab/EPICLab.github.io repository instead of having separate repositories for sub-pages. This also leads me to needing to migrate those two publication-specific pages into this repository (and redirecting the URLs to act appropriately based on that new structure).
